cosmoz-omnitable-column-mixin.html

```diff
@@ -0,0 +1,545 @@
+
+						cellClass: {
+							type: String,
+							value: 'default-cell'
+						},
+
+						headerCellClass: {
+							type: String,
+							value: 'default-header-cell'
+						},
+
+						styleModule: {
+							type: String
+						},
+
+						/**
+						 * Allow column to overflow without triggering
+						 * column folding (good for long descriptions)
+						 */
+						overflow: {
+							type: Boolean,
+							value: false
+						},
+
+						priority: {
+							type: Number,
+							value: 0
+						},
+
+						cellTemplate: {
+							type: Object,
+							observer: '_cellTemplateChanged'
+						},
+						/**
+						 * Cell title function, can be overriden.
+						 */
+						cellTitleFn: {
+							type: Function,
+							value() {
+								/**
+								 * @this item
+								 * @param {object} item Row item.
+								 * @param {string} valuePath Path in the item.
+								 * @returns {string} Cell title.
+								 */
+								return function (item, valuePath = this.valuePath) {
+									return this.getString(item, valuePath);
+								};
+							}
+						},
+
+						hidden: {
+							type: Boolean,
+							reflectToAttribute: true,
+							observer: '_hiddenChanged'
+						},
+						/**
+						 * Index of this column in the list of displayed columns (excluding disabled/hidden columns).
+						 */
+						columnIndex: {
+							type: Number
+						},
+
+						preferredDropdownHorizontalAlign: {
+							type: String,
+							computed: '_computePreferredDropdownHorizontalAlign(columnIndex)'
+						},
+
+						_dataTemplatizer: {
+							type: Object,
+							value: null
+						},
+
+						_headerTemplatizer: {
+							type: Object,
+							value: null
+						}
+					};
+				}
+
+				// StyleModules: [],
+
+				static get observers() {
+					return [
+						'__filterChanged(filter.*)',
+						'_pathsChanged(valuePath, groupOn, sortOn)'
+					];
+				}
+
+				ready() {
+					super.ready();
+					this._headerTemplatizer = this._createTemplatizer('template.header');
+					this._itemCells = [];
+				}
+
+				_createTemplatizer(templateElementOrSelector) {
+					var template,
+						templateHost,
+						templatizer,
+						root;
+					if (templateElementOrSelector instanceof HTMLTemplateElement) {
+						template = templateElementOrSelector;
+						templateHost = templateElementOrSelector.parentNode;
+					} else {
+						template = this.querySelector(
+							templateElementOrSelector
+						);
+						if (template) {
+							root = this.getRootNode();
+						} else {
+							root = this.getRootNode();
+							template = this.root.querySelector(templateElementOrSelector);
+						}
+						if (root) {
+							templateHost = root.host;
+						}
+					}
+
+					templatizer = document.createElement('cosmoz-omnitable-templatizer');
+					this.appendChild(templatizer);
+					templatizer.init(template, templateHost);
+					return templatizer;
+				}
+
+				_cellTemplateChanged(cellTemplate) {
+					if (cellTemplate !== undefined && cellTemplate !== null) {
```

Comment:
Only scenario where we use `==` is against null to capture both undefined and null:

```suggestion
					if (cellTemplate != null) {
```